### PR TITLE
feat: Zap

### PR DESCRIPTION
**Pull request by chef-jojo**

### CODE REVIEW

File: src/config/constants/index.ts [hunk 1]
```diff
@@ -72,6 +72,8 @@ export const DEFAULT_DEADLINE_FROM_NOW = 60 * 20
 
 export const BIG_INT_ZERO = JSBI.BigInt(0)
 
+export const LIQUIDLY_MINIMUM_AMOUNT = JSBI.BigInt(1000)
```
Comment (memoyil) [inline src/config/constants/index.ts:75]: typo

**Review by MirekBrzez — DISMISSED** [review]

**Comment by MirekBrzez** [comment]
Is everything Good, I'll be back with new phone.
Sorry

**Comment by memoyil** [comment]
It crashes when BNB to Bunny selected

**Comment by memoyil** [comment]
Imo if user has already lp's and clicks 'Add liquidity instead' button, it should bypass the choose valid pair page and directly points to the add liquidity page. wdyt @chef-jojo ?

**Comment by chef-jojo** [comment]
> Imo if user has already lp's and clicks 'Add liquidity instead' button, it should bypass the choose valid pair page and directly points to the add liquidity page. wdyt @chef-jojo ?

Added querystring to bypass choose pair 💪

**Comment by memoyil** [comment]
> > Imo if user has already lp's and clicks 'Add liquidity instead' button, it should bypass the choose valid pair page and directly points to the add liquidity page. wdyt @chef-jojo ?
> 
> Added querystring to bypass choose pair 💪

I think it should be vice versa, if user accesses via direct link or clicks get lp links in farms, it should be pointed to the add liquidity page. Imho valid pair page only needed when user wants to add liquidity from scratch in liquidity page

**Pull request merged**